!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  altitude_tolerance_error: 0.08
  altitude_when_red_gate: 1.6
  altitude_when_yellow_gate: 0.5
  avoidance_forward_time: 9.0
  blue_h_offset_x_tolerance: 9.0
  blue_h_offset_y_tolerance: 9.0
  forward_vel: 0.13
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      altitude_tolerance_error: 0.08
      altitude_when_red_gate: 1.6
      altitude_when_yellow_gate: 0.5
      avoidance_forward_time: 9.0
      blue_h_offset_x_tolerance: 9.0
      blue_h_offset_y_tolerance: 9.0
      forward_vel: 0.13
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      judge_achieve_counter_threshold: 30
      landing_altitude: 0.0
      name: Default
      normal_flight_altitude: 0.5
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      pid_p_v2p_x_has_threshold: true
      pid_p_v2p_x_kd: 1.4
      pid_p_v2p_x_ki: 0.0
      pid_p_v2p_x_kp: 1.2
      pid_p_v2p_x_threshold: 0.8
      pid_p_v2p_y_has_threshold: true
      pid_p_v2p_y_kd: 1.4
      pid_p_v2p_y_ki: 0.0
      pid_p_v2p_y_kp: 1.2
      pid_p_v2p_y_threshold: 0.8
      pid_p_v2p_yaw_has_threshold: true
      pid_p_v2p_yaw_kd: 0.7
      pid_p_v2p_yaw_ki: 0.0
      pid_p_v2p_yaw_kp: 0.3
      pid_p_v2p_yaw_threshold: 0.5
      pid_p_v2p_z_has_threshold: true
      pid_p_v2p_z_kd: 5.5
      pid_p_v2p_z_ki: 0.0
      pid_p_v2p_z_kp: 4.5
      pid_p_v2p_z_threshold: 0.75
      pid_v_v2p_x_has_threshold: true
      pid_v_v2p_x_kd: 0.007
      pid_v_v2p_x_ki: 0.0
      pid_v_v2p_x_kp: 0.004
      pid_v_v2p_x_threshold: 0.13
      pid_v_v2p_y_has_threshold: true
      pid_v_v2p_y_kd: 0.007
      pid_v_v2p_y_ki: 0.0
      pid_v_v2p_y_kp: 0.004
      pid_v_v2p_y_threshold: 0.13
      pid_v_v2p_yaw_has_threshold: true
      pid_v_v2p_yaw_kd: 1.2
      pid_v_v2p_yaw_ki: 0.0
      pid_v_v2p_yaw_kp: 1.0
      pid_v_v2p_yaw_threshold: 1.2
      pid_v_v2p_z_has_threshold: false
      pid_v_v2p_z_kd: 1.0
      pid_v_v2p_z_ki: 0.0
      pid_v_v2p_z_kp: 1.0
      pid_v_v2p_z_threshold: 1.0
      qr_offset_x_tolerance: 20.0
      qr_offset_y_tolerance: 20.0
      red_x_offset_x_tolerance: 9.0
      red_x_offset_y_tolerance: 9.0
      rotate_yaw_tolerance: 5.0
      state: true
      tf_break_duration: 0.2
      type: ''
    state: []
  judge_achieve_counter_threshold: 30
  landing_altitude: 0.0
  normal_flight_altitude: 0.5
  pid_p_v2p_x_has_threshold: true
  pid_p_v2p_x_kd: 1.4
  pid_p_v2p_x_ki: 0.0
  pid_p_v2p_x_kp: 1.2
  pid_p_v2p_x_threshold: 0.8
  pid_p_v2p_y_has_threshold: true
  pid_p_v2p_y_kd: 1.4
  pid_p_v2p_y_ki: 0.0
  pid_p_v2p_y_kp: 1.2
  pid_p_v2p_y_threshold: 0.8
  pid_p_v2p_yaw_has_threshold: true
  pid_p_v2p_yaw_kd: 0.7
  pid_p_v2p_yaw_ki: 0.0
  pid_p_v2p_yaw_kp: 0.3
  pid_p_v2p_yaw_threshold: 0.5
  pid_p_v2p_z_has_threshold: true
  pid_p_v2p_z_kd: 5.5
  pid_p_v2p_z_ki: 0.0
  pid_p_v2p_z_kp: 4.5
  pid_p_v2p_z_threshold: 0.75
  pid_v_v2p_x_has_threshold: true
  pid_v_v2p_x_kd: 0.007
  pid_v_v2p_x_ki: 0.0
  pid_v_v2p_x_kp: 0.004
  pid_v_v2p_x_threshold: 0.13
  pid_v_v2p_y_has_threshold: true
  pid_v_v2p_y_kd: 0.007
  pid_v_v2p_y_ki: 0.0
  pid_v_v2p_y_kp: 0.004
  pid_v_v2p_y_threshold: 0.13
  pid_v_v2p_yaw_has_threshold: true
  pid_v_v2p_yaw_kd: 1.2
  pid_v_v2p_yaw_ki: 0.0
  pid_v_v2p_yaw_kp: 1.0
  pid_v_v2p_yaw_threshold: 1.2
  pid_v_v2p_z_has_threshold: false
  pid_v_v2p_z_kd: 1.0
  pid_v_v2p_z_ki: 0.0
  pid_v_v2p_z_kp: 1.0
  pid_v_v2p_z_threshold: 1.0
  qr_offset_x_tolerance: 20.0
  qr_offset_y_tolerance: 20.0
  red_x_offset_x_tolerance: 9.0
  red_x_offset_y_tolerance: 9.0
  rotate_yaw_tolerance: 5.0
  tf_break_duration: 0.2
state: []
